### PR TITLE
nit: service misspelled in example

### PR DESCRIPTION
**Pull request by tobias-**

### CODE REVIEW

**Review by ikhoon — APPROVED** [review]
🙀

File: site/src/pages/docs/advanced-unit-testing.mdx [hunk 1]
```diff
@@ -125,7 +125,7 @@ executor.scheduleWithFixedDelay(() -> {
     HttpRequest req = HttpRequest.of(HttpMethod.POST, "/expire_stall_sessions");
     ServiceRequestContext ctx = ServiceRequestContext.of(req);
     try {
-        HttpResponse res = sessionManagementService.servce(ctx, req);
+        HttpResponse res = sessionManagementService.service(ctx, req);
```
Comment (minwoox) [inline site/src/pages/docs/advanced-unit-testing.mdx:128]: I think it's `serve` 😄

Comment (tobias-) [reply site/src/pages/docs/advanced-unit-testing.mdx:128]: haha, I'll fix it

**Review by minwoox — APPROVED** [review]
Thanks for finding this. 😄

**Review by trustin — APPROVED** [review]
Thanks, @tobias-!

**Pull request merged**